### PR TITLE
Save filters

### PR DESCRIPTION
**Pull request by cribeiro84**
Automatically save filters while the options are selected by the user.

### CODE REVIEW

File: src/components/FilterBarHub.tsx [hunk 1]
```diff
@@ -6,44 +6,54 @@ import { DropdownFilterBarItem } from "azure-devops-ui/Dropdown";
...
 import { IdentityRef } from "azure-devops-extension-api/WebApi/WebApi";
 import { getVoteDescription } from "./Columns";
 import { ITableColumn } from "azure-devops-ui/Table";
 import { Status } from "azure-devops-ui/Status";
 import { getStatusSizeValue, getStatusIcon } from "../models/constants";
+import { PullRequestModel } from "../models/PullRequestModel";
+import { Spinner } from "office-ui-fabric-react";
 
-export const myApprovalStatuses = Object.keys(Data.ReviewerVoteOption)
-  .filter(value => !isNaN(parseInt(value, 10)))
```
Comment (vladkasianenko) [inline src/components/FilterBarHub.tsx:29]: Why don't you rename this variable into something more meaningful?

File: src/tabs/PullRequestsTab.tsx [hunk 2]
```diff
@@ -549,6 +517,34 @@ export class PullRequestsTab extends React.Component<
...
+      }
+    } else {
+      this.storeFilterData(key, filterData);
+    }
+
+    return ;
+  }
+
+  private storeFilterData(key: string, data: any): void {
+    localStorage.setItem(`${STORED_CACHE_KEY_PREFIX}${key}`, JSON.stringify(data));
```
Comment (vladkasianenko) [inline src/tabs/PullRequestsTab.tsx]: What about introducing `const` for this value?

Comment (cribeiro84) [reply src/tabs/PullRequestsTab.tsx]: I'm not sure if I got your point. Which value do you refer?

File: src/components/FilterBarHub.tsx (hunk 1 above)
Comment (cribeiro84) [reply src/components/FilterBarHub.tsx:29]: Do you mean the myApprovalStatuses?

Comment (vladkasianenko) [reply src/components/FilterBarHub.tsx:29]: I mean `value` in the `filter` function.

**Pull request merged**